src/Corax/IndexSearcher.cs

```diff
@@ -557,6 +559,18 @@ public BoostingMatch Boost<TInner>(TInner match, IQueryScoreFunction scoreFuncti
             }
         }
 
+        public SuggestionTermProvider Suggest(int fieldId, string term, Analyzer analyzer = null, int take = -1)
+        {
+            // TODO: Until we figure out how to best call this through integration, I will add this convenience method. 
+            Slice.From(_transaction.Allocator, term, out var termSlice);
```

Comment:
Missing `using` ?

Reply:
No, we need it to survive. Either way, `string` based API will be removed in the future when we start optimizing and/or left only for non critical convenience usages (not a high traffic entry point by any means).